crates/types/src/config/admin.rs

```diff
@@ -173,6 +180,7 @@ impl Default for AdminOptions {
             concurrent_api_requests_limit: None,
             query_engine: Default::default(),
             heartbeat_interval: Duration::from_millis(1500).into(),
+            sync_interval: Duration::from_secs(10).into(),
```

Comment:
Let's call this `metadata_` to be more explicit:

```suggestion
            metadata_sync_interval: Duration::from_secs(10).into(),
```